kahuna/public/js/components/gr-image-persist-status/gr-image-persist-status.html

```diff
@@ -0,0 +1,9 @@
+<gr-icon-label ng:if="! ctrl.states.canArchive" gr-icon="lock"
+    title="Kept in library because: {{ctrl.states.persistedReasons}}">
```

Comment:
Could we build up nicer sentences here.
"Kept in library because of: exports" => "Kept in library because image has exports".

Also not sure how people feel about having the [help cursor](http://www.echoecho.com/csscursors.htm) on tootips as the title takes a smidgen of a second to appear.